### DISCUSSION TITLE
How to export/import playlists?

### DISCUSSION THREAD

**Discussion by james28909** [FreeTubeApp/FreeTube#3695] [Q&A]
![playlists](https://github.com/FreeTubeApp/FreeTube/assets/16311827/59955d94-a03d-4097-a2be-027b836f77fb)
when i export playlists from youtube, this is the output of the takeout.zip

when i try to import it, nothing shows up in the list, almost as if the file format isnt correct. i tried to change the file extension to .db but that didnt work either

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Hi @james28909, we currently have no way to import playlists from YT into FT. This is because we dont support the playlists functionality yet.
